doghouse/server/doghouse.go

```diff
@@ -34,18 +33,21 @@ const maxAnnotationsPerRequest = 50
 type Checker struct {
 	req              *doghouse.CheckRequest
 	gh               checkerGitHubClientInterface
+	cli              *github.Client
 	inDogHouseServer bool // If true, this checker runs in the DogHouse server.
 }
 
 func NewChecker(req *doghouse.CheckRequest, gh *github.Client, inDogHouseServer bool) *Checker {
 	return &Checker{
 		req:              req,
 		gh:               &checkerGitHubClient{Client: gh},
+		cli:              gh,
 		inDogHouseServer: inDogHouseServer,
 	}
 }
 
-func (ch *Checker) Check(ctx context.Context, makeRequest bool) (*doghouse.CheckResponse, error) {
+func (ch *Checker) Check(ctx context.Context) (*doghouse.CheckResponse, error) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method Checker.Check should have comment or be unexported
<!-- __reviewdog__:ChBlZTUxYjNkYTYzNTQwNjEzEg9nb2xpbnQtY2lyY2xlY2k= -->